test_scripts/Resumption/InteriorVehicleData/008_InteriorVD_resumption_same_moduleType_with_different_moduleIds_IGN_OFF.lua

```diff
@@ -0,0 +1,65 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle subscription for same moduleType with different moduleIds data
+--  after IGN_OFF
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 with modueId_1
+-- 4. App is subscribed to module_1 with modueId_2
+--
+-- Sequence:
+-- 1. IGN_OFF and IGN_ON are performed
+-- 2. App starts registration with actual hashId after SDL restart
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, module_1, moduleId_1) and
+--    RC.GetInteriorVehicleData(subscribe=true, module_1, moduleId_2) to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to mobile app
```

Comment:
please add 
b. SDL get 2 successful responses for request with isSubscribed=true
c. SDL respond RAI(success=true, result code = SUCCESS) to mobile app

Reply:
Done